test/shared_array_scope_list.rb

```diff
@@ -96,6 +106,11 @@ def test_insert_at
       new4.reload
       assert_equal 4, new4.pos
 
+      new = ArrayScopeListMixin.act_as_list_no_update do
+        ArrayScopeListMixin.create(parent_id: 20, parent_type: 'ParentClass')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.